cachalot/utils.py

```diff
@@ -105,18 +105,15 @@ def _find_subqueries_in_where(children):
             raise IsRawQuery
         elif child_class is NothingNode:
             pass
+        elif child_class is Subquery or child_class is Exists:
+            pass
```

Comment:
Can you rewrite `elif child_class is NothingNode:` to be combined? Something like:

```
elif child_class in (NothingNode, Subquery, Exists)
```